### PR TITLE
ci: update issue labeler

### LINKED ISSUE #2001: Fix issue labeler workflow
**Describe the bug**: 

Update labeler workflow to use the correct actions and groups, see nodejs changes:
https://github.com/elastic/apm-agent-nodejs/pull/3906/files

**To Reproduce**

1. Open an issue as @xrmx 
2. Issues labeled as community

**Environment (please complete the following information)**
- OS: [e.g. Linux]
- Python version:
- Framework and version [e.g. Django 2.1]:
- APM Server version:
- Agent version:

**Additional context**

None

### PR DESCRIPTION
**Pull request by xrmx**
## What does this pull request do?

Stop using the apm group and custom github action to check if issues are from internal members or not.

## Related issues

Closes #2001

### CODE REVIEW

File: .github/workflows/labeler.yml [hunk 1]
```diff
@@ -4,43 +4,29 @@ on:
     types: [opened]
   pull_request_target:
     types: [opened]
-env:
-  MY_GITHUB_TOKEN: ${{ secrets.APM_TECH_USER_TOKEN }}
+
+# 'issues: write' for https://docs.github.com/en/rest/issues/labels?apiVersion=2022-11-28#add-labels-to-an-issue
+permissions:
+  contents: read
+  issues: write
```
Comment (reakaleek) [inline .github/workflows/labeler.yml:11]: I think 
```suggestion
  issues: write
  pull-requests: write
```

I think we also need `pull-requests: write` if this is adding labels to PRs aswell.

Comment (xrmx) [reply .github/workflows/labeler.yml:11]: The documentation says that one should be enough. This conf is taken from the nodejs agent but didn't receive any community PR in the meantime. I would like to try with this one and add the other if something is off.

Comment (reakaleek) [reply .github/workflows/labeler.yml:11]: I was looking at https://docs.github.com/en/actions/using-jobs/assigning-permissions-to-jobs

image

Comment (reakaleek) [reply .github/workflows/labeler.yml:11]: But feel free to try it out.

**Review by reakaleek — APPROVED** [review]

**Pull request merged**